utils/index.js

```diff
@@ -115,8 +115,6 @@ export function visitAndClearCookies(account = ACCOUNT_NAMES.DEFAULT) {
 }
 
 export function getAddSkusEndpoint({ skus, account, salesChannel }) {
-  deleteAllCookies()
```

Comment:
In practice, what was the side-effect from this?

Reply:
it deleted the cookies before every `it` test cases, I want to run the e2e suite again to see if any of our tests actually relied on this

but in theory it shouldn't make a difference, because the `visitAndClearCookies` function always clear the cookies.. and it already runs before every test case as well

Reply:
I saw that this wasn't enough, if the tests fails the cypress dashboard will try again (up to 3 times) and on the second run it doesn't have the VTEX ID cookie anymore :sadge:

I'm thinking of moving the setCookie bit to `setup` instead, what do you think?

Reply:
> on the second run it doesn't have the VTEX ID cookie anymore :sadge:

oh :(

> I'm thinking of moving the setCookie bit to setup instead, what do you think?

makes sense, lgtm

Reply:
I actually updated the `before` to `beforeEach`, which I think makes more sense IMHO, but it resulted in a larger PR :-(